src/reducers/__tests__/index.test.js

```diff
@@ -0,0 +1,81 @@
+import { selectedMapPoint, ticketNum, SRSummary } from '../index';
+import {
+  SELECT_MAP_POINT,
+  UNSELECT_MAP_POINT,
+  SEARCH_TICKET_NUM,
+  SEARCH_TICKET_NUM_RESET,
+  RECEIVE_SR_SUMMARY,
+} from '../../actions/index';
+
+describe('selectMapPoint reducer', () => {
+  it('should return the initialState', () => {
+    const initialState = null;
+    const action = {};
+    const state = selectedMapPoint(undefined, action);
+    expect(state).toEqual(initialState);
```

Comment:
You can use jest matcher `.toBeNull()` instead of using `toEqual()` can read more [here](https://facebook.github.io/jest/docs/en/expect.html)